Cargo.toml

```diff
@@ -17,7 +17,7 @@ authors = ["Brooklyn Zelenka <hello@brooklynzelenka.com>"]
 [lib]
 path = "src/lib.rs"
 bench = false
-doctest = false
+doctest = true
```

Comment:
@expede now doctests will run as part of norm tests.

Reply:
Oooh gotcha. Nice 👍